Services/FlipperService.cs

```diff
@@ -184,7 +198,8 @@ public async Task ProcessSlowQueue()
                     if (SoldAuctions.ContainsKey(flip.UId))
                         flip.Sold = true;
                     NotifyAll(flip, SlowSubs);
-                    Console.Write("sf+" + SlowSubs.Count);
+                    if (flip.Uuid[0] == 'a')
```

Comment:
Why do you log all flips with a UUID starting with 'a'?
Only a notice, as I dont care what you log.

Reply:
This is just there to see if there still flips sent.
Its effectively just logging 1/16th of the flips to reduce the amount of logs that are stored